sycl/test-e2e/lit.cfg.py

```diff
@@ -37,6 +37,20 @@
 config.required_features = []
 config.unsupported_features = []
 
+# split-mode: Set if tests should run normally or with split build/run
+match lit_config.params.get("test-mode", "full"):
+    case "run-only":
+        config.test_mode = "run-only"
+        config.available_features.add("run-mode")
+    case "build-only":
+        config.test_mode = "build-only"
+        config.sycl_devices = []
+        arch_flag = ""
+    case _:
```

Comment:
`test-mode` can only accept `run-only`, `build-only`, and `full`, right? If so, shouldn't we do something like the following:
```
case "run-only":
   ...
case "build-only":
  ...
case "full":
  ...
case _:
  print("Invalid argument to test-mode")
  exit(1)
```

Reply:
ok, done in latest commit